oshmem/runtime/oshmem_shmem_params.c

```diff
@@ -42,15 +42,15 @@ int oshmem_shmem_register_params(void)
     (void) mca_base_var_register("oshmem",
                                  "oshmem",
                                  NULL,
-                                 "mcs_lock",
-                                 "Choose mcs locks (default = 0)",
+                                 "disable_mcs_lock",
```

Comment:
better keep it as "enable_mcs_lock". default value = 1. user can set to 0 to disable

Reply:
Sounds good. Modified now.